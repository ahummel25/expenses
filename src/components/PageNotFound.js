import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => (
    <div>
     <p>404! Page Not Found. <Link to="/">Go Home</Link></p>
    </div>
);

export default PageNotFound;