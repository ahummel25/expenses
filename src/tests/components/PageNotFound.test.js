import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test('should render PageNotFound', () => {
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find('p').length).toBe(1);
  expect(wrapper.find('p').text()).toContain('404! Page Not Found.');
});