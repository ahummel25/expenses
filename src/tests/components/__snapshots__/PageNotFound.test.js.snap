// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound 1`] = `
<div>
  <p>
    404! Page Not Found. 
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </p>
</div>
`;
